DataFormats/Detectors/TRD/include/DataFormatsTRD/RawData.h

```diff
@@ -62,79 +62,86 @@ Word 7  |link 13 datasize lower |   link 14 errorflags   |              link 14
         -------------------------------------------------------------------------------------------------
 
 alternate candidate :
-        |31|30|29|28|27|26|25|24|23|22|21|20|19|18|17|16|15|14|13|12|11|10|09|08|07|06|05|04|03|02|01|00|
+        |00|01|02|03|04|05|06|07|08|09|10|11|12|13|14|15|16|17|18|19|20|21|22|23|24|25|26|27|28|29|30|31|
```

Comment:
Is this bit numbering the usual one? I usually expect the lowest bits on the right. What is the RDH using?

In this numbering, I would kind of expect the bits in one byte to be reordered.